config/webpack/webpack.config.js

```diff
@@ -137,6 +138,9 @@ const webpackConfig = (options, env, argv) => {
       new webpack.DefinePlugin({
         CERNER_BUILD_TIMESTAMP: JSON.stringify(new Date(Date.now()).toISOString()),
       }),
+      new webpack.DefinePlugin({
+        SUPPORTED_LOCALES: JSON.stringify(supportedLocales),
```

Comment:
I may be paranoid, but i like to namespace things to be unique to us as possible. How about...
```suggestion
        TERRA_SUPPORTED_LOCALES: JSON.stringify(supportedLocales),
```

Reply:
That works for me: https://github.com/cerner/terra-toolkit/pull/383/commits/4bcd94ab3d5687509bed4dd444bdfd4cfce0eb70